tests/nosetests/pyanaconda_tests/module_source_nfs_test.py

```diff
@@ -115,9 +114,12 @@ def ready_state_test(self, ismount):
 
         self.assertTrue(self.source_module.is_ready())
```

Comment:
Btw. I think that you can remove most of the tests for the `is_ready` method and keep only one.

Reply:
Haha, I really am braindead :) Yes, the tests need way more changes, namely for the base classes. Sorry.

Reply:
I think I have addressed that by now...